packages/core-web/src/styles/page-nav.css

```diff
@@ -24,6 +24,10 @@
     white-space: inherit;
 }
 
+#page-nav > .nav-component > .navbar-brand.page-nav-title {
+    padding-left: 0.5rem;
```

Comment:
How about shifting this into the existing `.page-nav-title` block? Shouldn't require the first few selectors as there should only be one page-nav on every page.